packages/react-components/src/components/MessageThread.tsx

```diff
@@ -443,6 +446,10 @@ export type MessageProps = {
    * Message to render. It can type `ChatMessage` or `SystemMessage` or `CustomMessage`.
    */
   message: ChatMessage | SystemMessage | CustomMessage;
+  /**
+   * Strings from locale.
+   */
+  strings: ILocaleKeys;
```

Comment:
This type should be the strings needed for just MessageProps (probably a separate property per string needed so developers can use intellisense to know what strings this component needs)